protocol-configuration/mainnetAddresses.ts

```diff
@@ -1,6 +1,6 @@
-import { MainnetContractsConfig, AddressCategory } from '../types/types'; // imported without custom path to allow docs to autogen without ts errors
+import { AddressCategory } from '../types/types'; // imported without custom path to allow docs to autogen without ts errors
 
-const MainnetContractsConfig: MainnetContractsConfig = {
+export const MainnetContractsConfigs = {
```

Comment:
Converted this to a named export from a default export - generally better pattern unless there's a specific reason to use a default export.